src/store/getters.ts

```diff
@@ -166,5 +166,10 @@ export default {
   /** The entity registered office address. */
   getRegisteredOfficeAddress (state: StateIF): OfficeAddressIF {
     return state.registeredAddress
+  },
+
+  /** The custodians list. */
+  getCustodians (state: StateIF): any[] {
+    return []
```

Comment:
When this gets implemented, we might want to make sure it's always at least `[]` (never null) in the store. This will allow us to confidently check its length without conditional chaining (which doesn't work in the template anyway).
There are already some arrays in the store that are done this way. In my PR, I've done complianceWarnings this way as I think it makes for cleaner code.

Reply:
For sure, definitely agree! We can initialize it as an empty array. 
Will do this when we retrieve and apply the information to store :)